examples/deadline/All-In-AWS-Infrastructure-Basic/ts/bin/config.ts

```diff
@@ -15,11 +15,11 @@ import { MongoDbSsplLicenseAcceptance } from 'aws-rfdk';
 class AppConfig {
 
   /**
-   * A map of regions to Deadline Client Linux AMIs. As an example, the Linux Deadline 10.1.11.5 AMI ID from us-west-2
+   * A map of regions to Deadline Client Linux AMIs. As an example, the Linux Deadline 10.1.12.1 AMI ID from us-west-2
```

Comment:
Maybe we should also remove deadline 10.1.11.5 from `aws-rfdk/integ/scripts/bash/rfdk-integ-e2e.sh.`

Reply:
Updated